benchmark/utils/utils.py

```diff
@@ -194,3 +194,12 @@ def test(model, loader, device, hetero, progress_bar=True,
             total_examples += batch_size
             total_correct += int((pred == batch.y[:batch_size]).sum())
     return total_correct / total_examples
+
+
+@contextmanager
+def xpu_profiler(export_chrome_trace=True):
```

Comment:
An analogous pytorch profiler function is located in the [torch_geometric/profile/profile.py](https://github.com/pyg-team/pytorch_geometric/blob/376786f91f691eb2e98dca263029dac4a2963808/torch_geometric/profile/profile.py). I wonder if the one for xpu should also be there.

Reply:
Good idea 😉 Done.